web-app/src/cache/users/actions.js

```diff
@@ -72,11 +73,23 @@ export const login = (token) => (
       return true
     } catch (error) {
       dispatch(removeLoggedUser())
-      await swal({
-        title: i18n.t('errorMessages.failedLogin'),
-        text: error.message,
-        icon: 'error',
-      })
+      if (error.message === upvsForbiddenSubstitutionError) {
+        await swal({
+          title: i18n.t('errorMessages.failedLogin'),
+          text: error.message,
+          icon: 'error',
+          button: i18n.t('topBar.logout'),
+        }).then(() => {
+          // TODO: make it nicer
+          window.location = getLogoutUrl(token)
```

Comment:
maybe it would be nicer if we save token to `localStorage` and then call `getLogoutUrl()`, so logout flow is always the same